foundry.toml

```diff
@@ -15,15 +15,10 @@ fail_on_revert = true
 [profile.default.fmt]
 wrap_comments = true
 
-
 [profile.build]
 test = "/dev/null"
 script = "/dev/null"
 
-
-[profile.test]
-via_ir = false
-
```

Comment:
I still don't understand this change:
- it slows down tests and removes the possibility of speeding them up
- there is a dedicated issue to talk about the setup, but we're introducing a change related to it in this PR

Reply:
it's the reason why the adaptive curve irm has been deployed without via-ir (and makes loose one thousand gas to all transactions on blue).
waiting for a proper discussion on it, i move to the safest (but slower stuff) is that ok ?

Reply:
> it's the reason why the adaptive curve irm

i don't get this?

The reason the adaptive curve irm was deployed without IR is because the default profile was compiling without IR (and only the build profile was compiling via IR), not due to the test profile not compiling via IR

Reply:
> waiting for a proper discussion on it, i move to the safest (but slower stuff) is that ok ?

i don't like this because in case we're waiting for a discussion on the topic i prefer not changing anything but we already talked about this when working on Blue and @QGarchery & you were in favor of changing things pre-discussion and this is the way we want forward so i approved

Reply:
>The reason the adaptive curve irm was deployed without IR is because the default profile was compiling without IR (and only the build profile was compiling via IR)

I mean it's because we don't compile the same way all the time

Reply:
ok then i revert, and we speak about it in the issue